packages/sandbox/ui/src/components/component-controls/print-value.ts

```diff
@@ -23,6 +23,14 @@ export const printValue = (value?: ReactElement<any> | string | number | symbol)
   }
 
   if (isFunction(value)) {
```

Comment:
I'm assuming value is an object if it's supposed to have keys, how come we check if it's a function?

Reply:
we basically just check if typeOf value == `function`, if so we can have the function keys

Reply:
But this works because React.FC is actually a class which is of type function, right? 
A pure function would not have any accessible keys to my knowledge. 

So this is assuming that a function is actually a class, I'm wondering if there is a safer way to do this

Reply:
Never-mind, although it's correct that it would be a class and not a function (Thanks javascript) we do safeguard by checking if the key exists at all so it's really not an issue and i got stuck in my brain